asaplib/descriptors/global_descriptors.py

```diff
@@ -242,3 +251,40 @@ def create(self, frame):
             raise ValueError('the size of the system is larger than the max_atoms of the CM descriptor')
         # notice that we return an empty dictionary for "atomic descriptors"
         return {'acronym': self.acronym, 'descriptors': self.cm.create(frame, n_jobs=8)}, {}
+
+
+class FingerprintDescriptor:
```

Comment:
The logic of ASAPXYZ class is that the create() function for the descriptors takes a frame as argument.

So in this case,
create(self, frame):
    compute_morgan(frame.info['smiles'])